### PR TITLE
CHT: Translated 360 strings

### PR DESCRIPTION
**Pull request by daihakken**

### CODE REVIEW

File: data/language/chinese_traditional.txt [hunk 1]
```diff
@@ -826,16 +826,16 @@ STR_0821    :12月
...
 STR_0829    :{SMALLFONT}{BLACK}視窗標題 - 拖曳此來移動本視窗
-STR_0830    :{SMALLFONT}{BLACK}Zoom view in
-STR_0831    :{SMALLFONT}{BLACK}Zoom view out
-STR_0832    :{SMALLFONT}{BLACK}Rotate view 90{DEGREE} clockwise
+STR_0830    :{SMALLFONT}{BLACK}放大檢視區域
+STR_0831    :{SMALLFONT}{BLACK}縮小檢視區域
+STR_0832    :{SMALLFONT}{BLACK}將檢視區域順時針旋轉90{DEGREE}
 STR_0833    :{SMALLFONT}{BLACK}暫停遊戲
-STR_0834    :{SMALLFONT}{BLACK}Disk and game options
+STR_0834    :{SMALLFONT}{BLACK}硬碟及遊戲設定Disk and game options
```
Comment (Gymnasiast) [inline data/language/chinese_traditional.txt]: I think you forgot to erase the English part here.

File: data/language/chinese_traditional.txt [hunk 2]
```diff
@@ -1452,17 +1452,17 @@ STR_1448    :Watching new ride being constructed
...
-STR_1453    :Enter name for this guest:
-STR_1454    :Can't name guest...
-STR_1455    :Invalid name for guest
-STR_1456    :{WINDOW_COLOUR_2}Cash spent: {BLACK}{CURRENCY2DP}
-STR_1457    :{WINDOW_COLOUR_2}Cash in pocket: {BLACK}{CURRENCY2DP}
-STR_1458    :{WINDOW_COLOUR_2}Time in park: {BLACK}{REALTIME}
+STR_1452    :遊客的名字
+STR_1453    :請輸入這位遊客的新姓名:
+STR_1454    :不能命名這位遊客...
+STR_1455    :不適合的遊客姓名Invalid name for guest
```
Comment (Gymnasiast) [inline data/language/chinese_traditional.txt]: I think you forgot to erase the English part here.

File: data/language/chinese_traditional.txt [hunk 3]
```diff
@@ -2233,39 +2233,39 @@ STR_2229    :Slope up to vertical
 STR_2230    :Vertical track
 STR_2231    :Holding brake for drop
 STR_2232    :Cable lift hill
-STR_2233    :{SMALLFONT}{BLACK}Park information
-STR_2234    :Recent Messages
+STR_2233    :{SMALLFONT}{BLACK}樂園資料
+STR_2234    :最近訊息Recent Messages
```
Comment (Gymnasiast) [inline data/language/chinese_traditional.txt]: I think you forgot to erase the English part here.

**Comment by Gymnasiast** [comment]
You're making a lot of progress! :+1: 
I noticed a few errors, I put line notes to mark them.

**Comment by daihakken** [comment]
Thanks for proof-reading for my translation :+1:

**Comment by Gymnasiast** [comment]
No problem, I like to make OpenRCT2 as well-translated as possible. :)

**Pull request merged**